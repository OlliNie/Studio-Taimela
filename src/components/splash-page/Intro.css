.intro{

  margin-top:1rem;
  font-style:italic;
  font-weight:500;

  @media(min-width: 420px){
    font-size:1.4rem;
  }
  @media(min-width: 520px){
    font-size:1.8rem;
    margin:0;
  
  }
 
  @media(min-width:900px){
    font-size:2.2em;
  } 
  @media(min-width:1100px){
    font-size: 1.8rem;
    
  }
  @media(min-width:1300px){
    font-size: 2.2rem;
  }
  @media(min-width:1600px){
    font-size: 2.9em;
  }
}
.intro li:last-child{
  margin-top:2rem;
}