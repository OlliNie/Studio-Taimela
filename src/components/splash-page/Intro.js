import React from 'react';
import { introLines } from '../../text/splashPage';
import styles from './Intro.css';

export default function Intro() {
  const intro = introLines.map((line, i) => <li key={i}>{line}</li>);
  return <ul className={styles.intro}>{intro}</ul>;
}
