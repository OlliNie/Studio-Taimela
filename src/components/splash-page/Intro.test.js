import React from 'react';
import { shallow } from 'enzyme';
import Intro from './Intro';

describe('Intro component', () => {
  it('renders introducion', () => {
    const wrapper = shallow(<Intro />);
    expect(wrapper).toMatchSnapshot();
  });
});
