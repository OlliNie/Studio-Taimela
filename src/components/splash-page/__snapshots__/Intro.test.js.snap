// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro component renders introducion 1`] = `
<ul
  className="intro"
>
  <li
    key="0"
  >
    I help tech companies with
  </li>
  <li
    key="1"
  >
    complex solutions to bring
  </li>
  <li
    key="2"
  >
    out their best
  </li>
  <li
    key="3"
  >
    Virpi Taimela
  </li>
</ul>
`;
